packages/users/src/users.spec.ts

```diff
@@ -20,38 +19,4 @@ describe('Users...', () => {
       expect(user).to.have.property('storageAuth');
     });
   });
-
-  describe('browser storage', () => {
-    // new identity storage
-    const browserStorage = new BrowserStorage();
-
-    it('create & authenticate & remove', async function () {
-      this.timeout(10000);
-      // create new Users with browser storage
-      const users = await Users.withStorage(browserStorage, config);
-      const identity = await users.createIdentity();
-      const storedIds = await browserStorage.list();
-      // identity should be added into storage
-      expect(storedIds).to.have.length(1);
-      // we create fresh Users instance from storage
-      // users should be loaded (authenticated) from stored identities
-      const usersFromStorage = await Users.withStorage(browserStorage, config);
-      const spaceUsers = usersFromStorage.list();
-      // we expect 1 user is signed in
-      expect(spaceUsers).to.have.length(1);
-      const spaceUser = spaceUsers.shift();
-      expect(spaceUser).to.have.property('token');
-      expect(spaceUser).to.have.property('identity');
-      expect(spaceUser).to.have.property('storageAuth');
-
-      // identity must be matching
-      expect(spaceUser?.identity.public.toString()).to.equal(identity.public.toString());
-
-      // remove user identity
-      await usersFromStorage.remove(identity.public.toString());
-      const storedIdentities = await browserStorage.list();
-      expect(usersFromStorage.list()).to.have.length(0);
-      expect(storedIdentities).to.have.length(0);
-    });
-  });
```

Comment:
@studna headsup, I moved this into the integrations tests folder, because the BrowserStorage doesn't currently exist in the node environment and that part of tests fails when testing against node. But integration tests run exclusively against the browser env, so it works there.

Also, lets try and have fast unit tests run inside individual packages and live/test integration tests can be moved into the integration_tests folder.

Reply:
Oh, I already did a FS experimental driver https://github.com/FleekHQ/space-sdk/pull/3 to fix that issue. But it still needs to mock that socket challenge

Reply:
Okay, I just reviews that branch. You can merge it so I can fix this up and merge this as well. Thanks